docs/tutorials/controlling-wedo2-motor.md

```diff
@@ -133,37 +133,54 @@ Please note that for small speed values (less than 20%) the motor will not respo
 ## Python example
 
 To use pyhton with the WeDO 2.0 we need a BLE library. Unfortunately BLE
-support in python is still quite imature but at least pybluez offers an extension
-for BLE:
-
-    pip install pybluez
-    pip install pybluez[ble]
-
-This didn't work in my EV3 but since the ble extension is in fact gattlib I went for
-it:
+support in python is still quite imature but there is at least one library that
+works in ev3dev - [link](https://bitbucket.org/OscarAcena/pygattlib) gattlib
 
     sudo apt-get install pkg-config libboost-python-dev libboost-thread-dev \
      libbluetooth-dev libglib2.0-dev python-dev
 
     pip install gattlib
 
-Please note that it takes A LOT of memory and around 2 hours time to install gattlib.
+This library is also used as an extension for a more known library, [link](https://pypi.python.org/pypi/PyBluez) pybluez
+so if you want a library for both bluetooth "Classic" and BLE this would be better:
+
+    pip install pybluez
+    pip install pybluez[ble]
+
+Unfortunately I couldn't make it work in my ev3dev system.
+
+Please note that it takes **a lot** of memory and around 2 hours to install gattlib.
 After some failures ("virtual memory exhausted: Cannot allocate memory") I finally
-succeeded extending my ev3dev swapfile to 1024 MB.
+succeeded extending my ev3dev swapfile to 1024 MB (please first check if you have
+enough free space in your SD card):
 
-This short python script makes the motor spin 2 second in each direction then stop:
+    sudo nano /etc/dphys-swapfile
```

Comment:
Yes, I was afraid of that. I started all over again from a new SD card wit latest image to assure everything is correct.